lib/msal-common/src/client/RefreshTokenClient.ts

```diff
@@ -133,7 +138,7 @@ export class RefreshTokenClient extends BaseClient {
      */
     private async executeTokenRequest(request: CommonRefreshTokenRequest, authority: Authority)
         : Promise<NetworkResponse<ServerAuthorizationTokenResponse>> {
-
+        const acquireTokenMeasurement = this.performanceClient.startMeasurement(PerformanceEvents.NetworkPerformance, request.correlationId);    
         const requestBody = await this.createTokenRequestBody(request);
```

Comment:
It would probably be good to put a measure inside `createTokenRequestBody` as well, since that does async pop operations, which could impact the overall time for this operation.